lib/galaxy/tools/imp_exp/export_history.py

```diff
@@ -26,6 +26,9 @@ def create_archive(export_directory, out_file, gzip=False):
     except Exception as e:
         print('Error creating history archive: %s' % unicodify(e), file=sys.stderr)
         return 1
+    finally:
+        if os.path.exists(export_directory):
+            shutil.rmtree(export_directory, ignore_errors=True)
```

Comment:
You don't need that with `ignore_errors=True`
```suggestion
        if os.path.exists(export_directory):
```
```suggestion
        shutil.rmtree(export_directory, ignore_errors=True)
```